models/articles-model.js

```diff
@@ -12,9 +12,32 @@ exports.selectArticle = (article_id) => {
 };
 
 exports.selectArticles = () => {
-  return db.query(`SELECT articles.author, articles.title, articles.article_id, articles.topic, articles.created_at,
+  return db
+    .query(
+      `SELECT articles.author, articles.title, articles.article_id, articles.topic, articles.created_at,
    articles.votes, articles.article_img_url, COUNT (comments.article_id) AS comment_count FROM articles JOIN comments ON articles.article_id = comments.article_id GROUP BY articles.article_id 
-   ORDER BY articles.created_at DESC;`).then(({ rows }) => {
-     return rows;
-   })
-}
+   ORDER BY articles.created_at DESC;`
+    )
+    .then(({ rows }) => {
+      return rows;
+    });
+};
+
+exports.selectComments = (article_id) => {
+  return db
+    .query(
+      `SELECT comments.* FROM comments JOIN articles ON comments.article_id = articles.article_id 
+       WHERE 
+       comments.article_id = $1
+       ORDER BY
+       comments.created_at DESC;`,
+      [article_id]
+    )
+    .then(({ rows }) => {
+      if (rows.length === 0) {
+        return Promise.reject({ status: 404, msg: "article does not exist" });
+      }
```

Comment:
Please consider that there are two scenarios where our query could return an empty array. The first is where the article doesn't exist, the second where the article does exist but has no comments. How can you make sure that only articles that don't exist are handled by your Promise.reject on line 38?